modules/aggregation-layers/src/utils/color-utils.js

```diff
@@ -29,6 +29,9 @@ export const defaultColorRange = [
 
 // Converts a colorRange array to a flat array with 4 components per color
 export function colorRangeToFlatArray(colorRange, ArrayType, defaultValue) {
+  if (colorRange.constructor === ArrayType) {
```

Comment:
If the intention is to support a flat array from the user, we should relax this check to be `Arraybuffer.isView` or even `Numer.isFinite(colorRange[0])`